content/docs/command-reference/commit.md

```diff
@@ -62,26 +64,26 @@ There are many cases where the last step is not desirable (for example rapid
 iterations on an experiment). The `--no-commit` option prevents the last step
 from occurring (on the commands where it's available), saving time and space by
 not storing unwanted <abbr>data artifacts</abbr>. The file hash is still
-computed and added to the DVC-file, but the actual data file is not saved in the
-cache. This is where the `dvc commit` command comes into play. It performs that
-last step (saving the data in cache).
+computed and added to the `dvc.lock` or `.dvc` file, but the actual data file is
+not saved in the cache. This is where the `dvc commit` command comes into play.
+It performs that last step (saving the data in cache).
 
 Note that it's best to avoid the last two scenarios. They essentially
-force-update the [DVC-files](/doc/user-guide/dvc-files-and-directories) and save
-data to cache. They are still useful, but keep in mind that DVC can't guarantee
-reproducibility in those cases.
+force-update the `dvc.lock` or `.dvc` files and save data to cache. They are
```

Comment:
Perfect 🙏

It was just a simple replacement of "DVC-file" -> "dvc.lock or/and .dvc files" in most places but I wanted you to figure it out 🙂